.Rbuildignore

```diff
@@ -3,3 +3,7 @@
 ^shiny\.gosling\.Rproj$
 ^\.Rproj\.user$
 ^js$
+^inst/WORDLIST$
```

Comment:
What is the point of putting `WORDLIST` in `inst`, if it is not added to a built package? Please remove it from `.Rbuildignore`, so it can be used for automated spell checks.